src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/merge_transpose_reorder.cpp

```diff
@@ -152,8 +152,17 @@ std::vector<InputShape> static_shapes = {
     InputShape{{}, {{1, 32, 16, 16}}},
 };
 
+#if defined(OPENVINO_ARCH_ARM)
+const ExpectedResult successfull_fuse_result{1, 1, 3};
```

Comment:
It seems the reason is an additional reorder appears in the graph because the Eltwise (bias?) is not fused into groupconvolution node

Reply:
Yes, `EltwiseAdd ` is not in the list of supported postops: https://github.com/alvoron/openvino/blob/37ab2c6abe2519e1ae2494e163798dfce7d8fce4/src/plugins/intel_cpu/src/dnnl_extension_utils.cpp#L227-L234